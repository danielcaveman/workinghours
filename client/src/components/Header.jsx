import React from "react";
import styled from "styled-components";

const HeaderContainer = styled.header`
  grid-area: header;
  background-color: #5eabf2;
`;

function Header() {
  return <HeaderContainer></HeaderContainer>;
}

export default Header;
