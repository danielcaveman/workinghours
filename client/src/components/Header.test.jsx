import React from "react";
import { render, screen } from "@testing-library/react";
import Header from "./Header";

describe("<Header />", () => {
  it("Should render the header component correctly", () => {
    render(<Header />);

    expect(screen.getByRole("banner")).toBeInTheDocument();
  });
});
