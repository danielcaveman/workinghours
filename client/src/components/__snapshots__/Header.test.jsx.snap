// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Should match snapshot 1`] = `
<div>
  <header
    class="sc-bdfBwQ QtUwF"
  />
</div>
`;
